### PR TITLE
Madsen/balance changes 3.8.7

### PR DESCRIPTION
**Pull request by Ripzss**
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2373
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2369
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2365
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2364
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2360
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2353
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2352
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2350
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2349
Closes https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2338

Reverted Food limit Reduction introduced in 3.8.0

### CODE REVIEW

**Comment by Ripzss** [comment]
Made it so Zandalari can now train their elites from Zul'farak

File: mapdata/WarcraftLegacies/UnitData/Skin/909455471.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Font of Arcane energy.|nProvides 10 food, which increases the maximum number of units that can be trained.|n|r|n|cff99b4d1Can only build 48."
+      "Value": "Font of arcane power. Provides 8 food, which increases the maximum number of units that can be trained.|n|cfff5962dAbilities:|R Charge Mana|n|n|cff99b4d1Can only train 48"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/UnitData/Skin/909455471.json:39]: This should still say "build", not "train". Buildings don't get trained.

File: mapdata/WarcraftLegacies/UnitData/Skin/859189352.json [hunk 2]
```diff
@@ -61,7 +61,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Font of arcane power. Provides 8 food, which increases the maximum number of units that can be trained.|n|cfff5962dAbilities:|R Charge Mana|n|n|cff99b4d1Can only train 48"
+      "Value": "Font of Arcane energy.|nProvides 10 food, which increases the maximum number of units that can be trained.|n|r|n|cff99b4d1Can only build 48."
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/UnitData/Skin/859189352.json:64]: This should still say "build", not "train". Buildings don't get trained.

Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/UnitData/Skin/859189352.json:64]: Dalaran's farms have no build limit, so the "Can only build 48" bit is incorrect.

Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/UnitData/Skin/859189352.json:64]: No need to capitalize arcane; it's not a proper noun.

**Comment by YakaryBovine** [comment]
There's a tonne of changes in here so it's a bit hard to review but I've done my best. Looks good, just still some issues with the Dalaran and Exodar Well tooltips. I know you didn't cause the issues I'm pointing out but that ticket can't be closed until the tooltip is actually correct.

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
There's a tonne of changes in here so it's a bit hard to review but I've done my best. Looks good, just still some issues with the Dalaran and Exodar Well tooltips. I know you didn't cause the issues I'm pointing out but that ticket can't be closed until the tooltip is actually correct.

**Review by YakaryBovine — APPROVED** [review]
Looking good!

**Pull request merged**